index.html

```diff
@@ -301,8 +293,8 @@ <h1 class="contact-head-me">Contact me</h1>
             </div>
             <form action="https://formspree.io/f/xnqrezql" method="post" >
               <ul class="form">
-                <li id="name-text-area">
-                  <div id="name-basic-container"><input type="text" name="name" placeholder="Enter Full Name" maxlength="30" class="name-basic-container name-text-input" required />
+                <li id="name-basic-container">
+                  <div id=""><input type="text" name="name" placeholder="Enter Full Name" maxlength="30" class="name-text-area name-text-input" required />
                   </div>
                 </li>
```

Comment:
- [ ] I noticed that the input fields in your contact form on the mobile version overflowed out of their containers. Kindly fix this, please.
![1](https://user-images.githubusercontent.com/94551484/208216309-5f5afa3b-ab2f-4548-b098-9d41228b9cf4.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/208216356-3a0b7b91-89fc-4e11-9f54-582c4ae4ca4c.png)